### ISSUE TITLE
Padding between action button

### ISSUE THREAD

**Issue by Mylloon** [whitescent/Mastify#70]
Maybe adding space between the comment, like, and boost button, instead of being all to the left may be a good idea?

**Comment by whitescent** [comment]
there's actually some padding between them, just a little bit, do you mean a spacing like Twitter?

![image](https://github.com/whitescent/Mastify/assets/31311826/6328e2e4-b4f6-473d-ae9b-f4d0dba9a82d)

**Comment by Mylloon** [comment]
Yes, something like Space Around or Space Evenly, I don't know which one is the best
